elide-datastore/elide-datastore-hibernate3/src/test/resources/logback.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2015, Yahoo Inc.
+  ~ Licensed under the Apache License, Version 2.0
+  ~ See LICENSE file in project root for terms.
+  -->
+<configuration>
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <filter
+                class="ch.qos.logback.classic.filter.ThresholdFilter">
+            <level>INFO</level>
+        </filter>
+        <encoder>
+            <pattern>%msg%n</pattern>
+        </encoder>
+    </appender>
+    <appender name="FILE" class="ch.qos.logback.core.FileAppender">
+        <file>target/trace.log</file>
+        <append>false</append>
+        <encoder>
+            <pattern>%msg%n</pattern>
+        </encoder>
+    </appender>
+    <appender name="ASYNC" class="ch.qos.logback.classic.AsyncAppender">
+        <appender-ref ref="FILE" />
+        <neverBlock>true</neverBlock>
+    </appender>
+
+    <logger name="com.yahoo.elide" level="TRACE" />
+
+    <root level="INFO">
+        <appender-ref ref="STDOUT" />
```

Comment:
We don't want the build to output more log lines to the console.  Travis enforces limits (which we have hit in the past).

Reply:
Do we really need this file?  Can we do without?

Reply:
We used this file for testing. If we do not include the files the log will not show up in STDOUT or the log file (target/trace.log here)

Reply:
We can remove logging from the console and only to the given file?